### DISCUSSION TITLE
Is there a "satellite map" offline available? (e. g. ESRI Satellite)

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#562] [Q&A] [answered]
In the middle of nowhere it would be great to get an overview of the surrounding tracks like this "online" possibility:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/25821704-5a17-4754-8668-a98b64b49255"/>   <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/769ff8a9-abd7-46ca-83a2-38aa73ed965a"/>
But often in the middle of nowhere there is no internet connection either and we have to live with this:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/40e294bb-d040-497d-bedb-e6804d66d3fb"/>   <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/304dcae1-723b-4bd8-ae1a-6e03ebd125c7"/>
Any ideas for me?

**Comment by devemux86** [comment, marked as answer]
@WolfgangZ42 thanks for the question.

Satellite maps are not vector maps (with very small size) like the Mapsforge map files.
They are bitmap tile maps, their size could be huge compared to the Mapsforge maps.

That's why they are provided as an online map, like the other online maps.

**Reply by DIrgi** [reply]
Additionally....every zoom level requires a new map tile download means as devmux mentioned, hell of download volume.
When using vector graphics you (I assume) download only ones the maximum possible vector data and map information.
When you zoom and navigate, it's just a new recalculation and (depending on your zoom level) adding or deleting some more map details on your hardware and no download.
For that reason, vector maps are (even when we talk about some gigabytes for complete countries) rather small in comparison to pixel graphics.
